vendor/native_mate/native_mate/wrappable.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE.chromium file.
+
+#include "native_mate/wrappable.h"
+
+#include "base/logging.h"
+#include "native_mate/dictionary.h"
+#include "native_mate/object_template_builder.h"
+
+namespace mate {
+
+WrappableBase::WrappableBase()
+    : isolate_(nullptr) {
+}
+
+WrappableBase::~WrappableBase() {
+  if (!wrapper_.IsEmpty())
+    GetWrapper()->SetAlignedPointerInInternalField(0, nullptr);
+  wrapper_.Reset();
+}
+
+v8::Local<v8::Object> WrappableBase::GetWrapper() {
+  if (!wrapper_.IsEmpty())
```

Comment:
this is the change to native_mate from https://github.com/electron/native-mate/blob/master/native_mate/wrappable.cc#L27